learningobjects/tests/test_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Test api functions.
+"""
+
+from django.test.testcases import TestCase
+
+from django.contrib.auth.models import User
+
+from learningobjects.models import Course
+from learningobjects.api import create_course
+
+# pylint: disable=no-member
```

Comment:
It happens with everything Django -- no model has member `objects`, nor `get_or_create` or `create`.

Also, it happens with `lxml.etree`.

I've kept them local when possible, but in files where models are used from multiple places I put them at the top. My logic is that if something actually _doesn't_ have a member the code will be broken and tests will fail anyway, so the warning is pretty worthless.